nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/QueryMarkLogic.java

```diff
@@ -198,11 +198,19 @@ protected Collection<ValidationResult> customValidate(final ValidationContext va
      * @throws ProcessException
      */
     @Override
+    // TODO: Ask Rob how to pass the flow file to success areas.
```

Comment:
This is already happening via `buildQueryBatchListener` - the tests should already verify this.